src/Project/Classes/Stores/Products/Instrument.java

```diff
@@ -0,0 +1,81 @@
+package Project.Classes.Stores.Products;
+
+public class Instrument extends Product {
+    private int yearCreated;
+    private int warranty;
+    private String manufacturer;

```

Comment:
[Fix] vezi comentariile de la clasa accessory, si aici si acolo ai niste campuri care clar apartin in clasa parinte, si nu dublate in fiecare clasa copil